mapper/orm/src/main/java/org/hibernate/search/mapper/orm/logging/impl/Log.java

```diff
@@ -131,9 +131,11 @@ SearchException errorRetrievingPropertyTypeModel(String propertyModelName,
 	SearchException hibernateSessionIsClosed(@Cause IllegalStateException cause);
 
 	@Message(id = ID_OFFSET + 18,
-			value = "Invalid automatic indexing synchronization strategy name: '%1$s'. Valid names are: %2$s.")
-	SearchException invalidAutomaticIndexingSynchronizationStrategyName(String invalidRepresentation,
+			value = "Invalid entity loading cache lookup strategy name: '%1$s'. Valid names are: %2$s.")
+	SearchException invalidEntityLoadingCacheLookupStrategyName(String invalidRepresentation,
 			List<String> validRepresentations);
+	//TODO: Yoann, ^ is it ok to update the message like this, or should we create a new one, and just delete this one
+	// so that ids will be pointing to the same messages, no matter the version ?
```

Comment:
Good catch... I think at this point it'll be a mess whatever we do; your solution looks good to me.